### DISCUSSION TITLE
Metody do pobierania współczynników jakości konta

### DISCUSSION THREAD

**Discussion by kfijalko** [allegro/allegro-api#4399] [Suggestions - in progress]
Czy planujecie udostępnienie metod do pobierania współczynników jakości konta?

**Comment by PrzemyslawLukanowski** [comment]
Na tę chwilę nie mamy tego w planach. Poinformujemy, jeśli coś się zmieni w tym temacie.

**Reply by TomaszSzydlowski-sudo** [reply]
@PrzemyslawLukanowski dlaczego odpisujecie wybiórczo na sugestie? Dałem sugestie dwa tygodnie temu i do teraz nie mam pojęcia czy ktoś w ogóle to zauważył.

**Reply by PrzemyslawLukanowski** [reply]
Jeżeli sam założyłeś wątek na Discussions i zgłosiłeś nam sugestię - nie odpisujemy na nią. Zgodnie z naszym [newsem](https://github.com/allegro/allegro-api/issues/4337) użytkownicy mogą na nią głosować, a jeżeli zdecydujmy wprowadzić zaproponowane przez Ciebie rozwiązanie, zaktualizujemy status sugestii. Część sugestii posiada już odpowiedzi, ponieważ przenieśliśmy je z zakładki "Issues", gdzie odpowiadamy na każdy wątek. W tym konkretnym przypadku kfijałko prócz tego, że zgłosił sugestię, dodatkowo zapytał wprost, czy planujemy wprowadzić zmianę w najbliższej przyszłości - stąd odpowiedź w zgłoszeniu.
